packages/webpack-cli/lib/plugins/CLIPlugin.js

```diff
@@ -52,7 +60,13 @@ class CLIPlugin {
                 this.logger.warn('You are using "bail" with "watch". "bail" will still exit webpack when the first error is found.');
             }
 
-            this.logger.log(`Compilation${getCompilationName()} starting...`);
+            const name = getCompilationName();
+
+            this.logger.log(`Compiler${name ? ` ${name}` : ''} starting...`);
+
+            if (configPath) {
+                this.logger.log(`Compiler${name ? ` ${name}` : ''} is using config: '${configPath}'`);
+            }
```

Comment:
I think we was wrong when we use `compilation` word here, we even use hooks for `compiler`